specs/protocol/stage-1.md

```diff
@@ -32,11 +34,118 @@ Within the context of an OP Stack, the following roles are required for Stage 1:
    This includes upgrading L1 contracts, modifying the implementation of the dispute game, and
    any other safety-critical functions.
 
-2. **Guardian:** This account MUST control any action which may cause a delay in the finalization of
+2. **Guardian:** This account MUST control any action which may delay a users ability to
+   exit the system to L1. Such delays may result from anything that impacts the finalization of
    L2 states and the resulting settlement on L1.
 
    This includes but is not limited to pausing code paths related to withdrawals.
 
 There may be additional [roles](./configurability.md#admin-roles) in the system, however they MUST
 not be able to perform any actions which have an impact on either the validity of L2 states, or the
 users ability to exit the system.
+
+## Configuration of Safes
+
+This list outlines the various Safes, their extensions, and other configuration details necessary to
+achieve Stage 1.
+
+1. **The L1 ProxyAdmin Owner Safe:** This Safe acts as the Upgrade Controller. Accordingly, it is
+   authorized to call the following functions:
+      - All `ProxyAdmin` `onlyOwner` functions.
+      - All `DisputeGameFactory` `onlyOwner` functions.
+      - All `DelayedWETH` `onlyOwner` functions.
+
+   This safe has a threshold of 2, and is owned by two other Safes:
+      1. The Security Council Safe.
+      2. The Optimism Foundation Upgrades Safe.
+
+   In general, the threshold and number of owners MUST be configured such that an upgrade
+   or other safety-critical action can NOT be performed without the cooperation of the Security Council.
+
+1. **The Guardian Safe:** This Safe controls any code paths which impact a users ability to
+   exit the system. It is extended with the
+   [Deputy Guardian Module](./safe-extensions.md#deputy-guardian-module).
+
+
+   Accordingly, this Safe is authorized to call the following liveness-critical functions:
```

Comment:
I would say `setAnchorState` is a safety function, since I believe it's used if a game resolved incorrectly

Reply:
b2eb20ade9d442b269d733569a02fde13bedee8e